README.md

```diff
@@ -35,23 +38,27 @@ MetaCompilation
 
 The MetaCompilation Analyzer is an analyzer that functions as a tutorial to teach users how to write an analyzer. It uses diagnostics and code fixes to guide the user through the various steps required to create a simple analyzer. It is designed for a novice analyzer programmer with some previous programming experience.
 
-For instructions on using this tutorial, see [Instructions](https://github.com/dotnet/roslyn-analyzers/blob/master/src/MetaCompilation/MetaCompilation/MetaCompilation/ReadMe.md#instructions).
+For instructions on using this tutorial, see [Instructions](src/MetaCompilation/MetaCompilation/MetaCompilation/ReadMe.md#instructions).
 
 
 Microsoft.AnalyzerPowerPack
 ---------------------------
 
+*Latest stable version:* [1.0.1](https://www.nuget.org/packages/Microsoft.AnalyzerPowerPack/)
+
 General language rules implemented as analyzers using the .NET Compiler Platform ("Roslyn").
 
-*Latest stable version:* [1.0.1](https://www.nuget.org/packages/Microsoft.AnalyzerPowerPack/)
+[More info](src/AnalyzerPowerPack/AnalyzerPowerPack.md)
 
 
 Microsoft.CodeAnalysis.Analyzers
 --------------------------------
 
+*Latest stable version:* [1.0.0](https://www.nuget.org/packages/Microsoft.CodeAnalysis.Analyzers/)
+
 Provides guidelines for using .NET Compiler Platform ("Roslyn") APIs.
```

Comment:
Could you add something like: "Use these analyzers to help you write your own Roslyn analyzers, fixers, and refactorings." I know that's implicit in "using the .NET Compiler Platform ("Roslyn") APIs" but still, for some reason, whenever I see this, I have to remind myself that's what it's for. Mostly